### PR TITLE
add setLiquiditySourceLsps2

### PR DESCRIPTION
**Pull request by BitcoinZavior**

### CODE REVIEW

**Comment by kumulynja** [comment]
`setLiquiditySourceLsps2` is working now 👍 
But `receiveViaJitChannel` and `receiveVariableAmountViaJitChannel` are missing to be able to use JIT channels completely.

**Comment by BitcoinZavior** [comment]
> receiveVariableAmountViaJitChannel

Please check now

File: src/classes/Node.ts [hunk 1]
```diff
@@ -250,6 +250,16 @@ export class Node extends NativeLoader {
     return await this._ldk.receiveViaJitChannel(this.id, amountMsat, description, expirySecs);
   }
 
+  /**
+   * Returns a payable invoice that can be used to request and receive a payment for which the amount is to be determined by the user, also known as a "zero-amount" invoice.
+   * @requires [description]
+   * @requires [expirySecs] number
+   * @returns {Promise<boolean>}
+   */
+  async receiveViaJitChannel(amountMsat: number, description: string, expirySecs: number): Promise<string> {
```
Comment (kumulynja) [inline src/classes/Node.ts:255]: `receiveViaJitChannel` was already available in the file and class, see line 249. So I think this has to be removed again.

File: src/classes/NativeLoader.ts [hunk 2]
```diff
@@ -31,9 +31,6 @@ export interface NativeLdkNodeRn {
   build(buildId: string): string;
 
   setLiquiditySourceLsps2(address: string, publicKey: string, token: String, buildId: string): any;
-  bolt11Payment(nodeId: string): any;
```
Comment (kumulynja) [inline src/classes/NativeLoader.ts:34]: `bolt11Payment` didn't had to be removed I think. So should be added back.

**Review by kumulynja — CHANGES_REQUESTED** [review]

File: src/classes/Node.ts (hunk 1 above)
Comment (BitcoinZavior) [reply src/classes/Node.ts:255]: Done

File: src/classes/NativeLoader.ts (hunk 2 above)
Comment (BitcoinZavior) [reply src/classes/NativeLoader.ts:34]: Done

**Pull request merged**